derive_builder_core/src/setter.rs

```diff
@@ -152,6 +164,46 @@ impl<'a> ToTokens for Setter<'a> {
     }
 }
 
+// adapted from https://stackoverflow.com/a/55277337/469066
+// Note that since syn is a parser, it works with tokens.
+// We cannot know for sure that this is an Option. The user could, for example, type
+// - `std::option::Option`
+// - `type MaybeString = std::option::Option<String>`
+// We cannot handle those arbitrary names.
```

Comment:
I agree that we can't handle arbitrary aliases, but is there a reason we can't handle `std::option::Option`, `::std::option::Option` (and the same for `core`)? I think that would improve our coverage to almost 100% (because who redefines/aliases `Option`?)

Reply:
I though the same, I'll add string comparison for those cases.

Reply:
it's more complicated than string comparison, it will take more time than expected.